### PR TITLE
Add ARC assessment result upload test to Exporter3Test

### PR DESCRIPTION
**Pull request by nategbrown9**

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java [hunk 1]
```diff
@@ -847,6 +884,160 @@ public void uploadWithScheduleContext() throws Exception {
...
+        participantsApi.updateParticipant(userId, participant).execute();
+
+        // Make Open Bridge survey assessment.
+        String assessmentId = "symbol_test";
+
+        // Get ARC symbol test
+        Assessment assessment = assessmentsApi.getAssessmentById(assessmentId, 1L).execute().body();
+
+        // To run locally you will need to create a symbol_test assessment
+        //if (assessment == null) {
```
Comment (Erin-Mounts) [inline src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java]: TODO or remove commented out code

**Review by Erin-Mounts — APPROVED** [review]
One nit, otherwise  looks good to me

File: src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java (hunk 1 above)
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java]: The assessments created by this test are deleted in after(), so we'll need to create them every time.

If we want to have assessments that stick around across integ test runs, we'll want to add them to https://github.com/Sage-Bionetworks/BridgeIntegrationTests/blob/develop/src/test/java/org/sagebionetworks/bridge/sdk/integration/InitListener.java (This runs in the Server Integ Tests.) Long term, we're going to use the BridgeStackBuilder (under construction) to boot strap test environments.

**Review by DwayneJengSage — APPROVED** [review]
Looks good. One comment about the commented out assessment.

File: src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java (hunk 1 above)
Comment (DwayneJengSage) [reply src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java]: As per Wednesday's meeting, these tests don't run on Prod, so we don't worry about cluttering up Prod with test assessments.

In Dev/Staging, we can either create and delete the assessment with every test run, or we can check whether the assessment already exists and create it if it doesn't, which should leave a singleton assessment that persists across test runs. (Your commented out code already does this.)

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**